sgx-isa/src/lib.rs

```diff
@@ -22,10 +23,10 @@ extern crate std;
 
 #[macro_use]
 extern crate bitflags;
-
 #[cfg(all(feature = "sgxstd", target_env = "sgx"))]
 use std::os::fortanix_sgx::arch;
-
+#[cfg(all(feature = "nightly", target_env = "sgx"))]
```

Comment:
If you define both `sgxstd` and `nightly`, you'll get a namespace conflict. You should make it such that `sgxstd` takes precedence.